doc/replicasets.rst

```diff
@@ -0,0 +1,226 @@
+.. _cartridge-cli.replicasets:
+
+===============================================================================
+Setting up replicasets
+===============================================================================
+
+``cartridge replicasets`` command is used to set up replicasets on local
+development.
+
+-------------------------------------------------------------------------------
+Usage
+-------------------------------------------------------------------------------
+
+.. code-block:: bash
+
+    cartridge replicasets [command] [flags] [args]
+
+All ``replicasets`` sub-commands has these flags:
+
+* ``--name`` - application name
+* ``--run-dir`` - directory where PID and socket files are stored
+  (defaults to ./tmp/run or "run-dir" in .cartridge.yml)
+* ``--cfg`` - configuration file for instances
+  (defaults to ./instances.yml or "cfg" in .cartridge.yml)
+
+-------------------------------------------------------------------------------
+How does it work?
+-------------------------------------------------------------------------------
+
+Replicasets are configured via instance console sockets placed in the run directory
+using Cartridge Lua API.
+All topology instances should be described in ``instances.yml`` file (see ``--cfg``).
+
+First, all running instances mentioned in ``instances.yml`` are connected to membership.
```

Comment:
membership module?

Reply:
No, membership is a group of instances (module has the same name, but in fact membership module connects instances to membership).